src/Dfe.PlanTech.Application/Response/Queries/GetLatestResponseListForSubmissionQuery.cs

```diff
@@ -30,5 +30,20 @@ public async Task<List<QuestionWithAnswer>> GetLatestResponseListForSubmissionBy
 
             return await _db.ToListAsync(responseListByDate);
         }
+
+        public async Task<List<QuestionWithAnswer>> GetResponseListByDateCreated(int submissionId)
+        {
+            var responseListByDate = _db.GetResponses
+                            .Where(response => response.SubmissionId == submissionId)
+                            .Select(response => new QuestionWithAnswer()
+                            {
+                                QuestionRef = response.Question.ContentfulRef,
+                                AnswerRef = response.Answer.ContentfulRef,
+                                DateCreated = response.DateCreated
+                            })
+                            .OrderByDescending(questionWithAnswer => questionWithAnswer.DateCreated);
```

Comment:
Is the `DateCreated` field being used anywhere else? Otherwise it would be more efficient to do:
- Where
- OrderByDescending
- Select

And not return the `DateCreated` field at all.